src/editor.js

```diff
@@ -223,25 +225,33 @@ const FileListItem = ({name}) => html`
  * of it, to propagate a click into a hidden `input[type=file]` element.
  * @param {Event} e
  */
-function cascasdeInputClick({currentTarget}) {
+function cascadeInputClick({currentTarget}) {
   assert(currentTarget.querySelector('input')).click();
 }
 
 const dispatchUploadFiles = redispatchAs(g('upload-files'));
 
+const dispatchShowTemplates = redispatchAs(g('show-templates'));
+
 /**
  * Renders a button to "upload" files--that is, set Blob URLs so they're
  * accessible from the AMP Ad document.
  */
 const FileUploadButton = () => html`
-  <div class="${n('upload-button-container')}" @click="${cascasdeInputClick}">
+  <div class="${n('upload-button-container')}" @click="${cascadeInputClick}">
     <div class="${n('upload-button')}">
       Add files
     </div>
     <input type="file" hidden multiple @change="${dispatchUploadFiles}" />
   </div>
 `;
 
+const ChooseTemplatesButton = () => html`
+  <div class="${n('upload-button')}" @click=${dispatchShowTemplates}>
```

Comment:
👮🚨👮🚨👮 `upload-button` abuse. 👮🚨👮🚨👮

Let's rename `upload-button` styles to a generic `text-button` and add that class to `UploadButton` accordingly.